recipe/meta.yaml

```diff
@@ -1,33 +1,62 @@
+{% set version = "1.2.7" %}
+
+{% if cuda_compiler_version == "None" %}
+{% set cuda_major = 0 %}
+{% else %}
+{% set cuda_major = environ.get("cuda_compiler_version", "11.8").split(".")[0] | int %}
+{% endif %}
+
 package:
   name: catboost
-  version: "1.2.7"
+  version: {{ version }}
+
+source:
+  url: https://github.com/catboost/catboost/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 099ff437993621b86565f3b25150071bdc20623ff77b75ed4d0cf6a37263e1fd
+  patches:
+    - conda.diff
 
 build:
-  number: 0
+  number: 1
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
-  # - OS X: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
   # - Win-64: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
   skip: True  # [win32]
   skip: True  # [python_impl == 'pypy']
-  # There are issues with python 3.8 on macOS 11.x: https://github.com/conda-forge/python-feedstock/issues/445
-  #  so Conda has issues with tags compatibility with platform tags 'macosx_11_0_...'
-  #  so disable this configuration for now
-  skip: True  # [osx and x86_64 and (py == 38)]
   # TODO: support Linux on ppc64le: https://github.com/catboost/catboost/issues/2145
   skip: True  # [ppc64le]
+  skip: True  # [linux and cuda_compiler_version == "None"]
```

Comment:
I'm going with the pypi strategy where we only have CUDA builds.

Reply:
> With this skip you are telling conda that catboost won't work if you don't have CUDA, which is wrong.
I can't find enough info whether it will actually prevent installation in environments where CUDA is not available but semantically it sends this message.

It doesn't tell conda that it won't work if we don't have CUDA. It just tells conda-build (which is separate from conda) to not build a package without CUDA.

> Why not do the thing conda expects us to do when cuda_compiler_version == "None" - build the package without CUDA support?

I'll add a commit that adds a package without CUDA support.